src/bin/pg_autoctl/keeper.c

```diff
@@ -1075,10 +1079,11 @@ keeper_maintain_replication_slots(Keeper *keeper)
 		return true;
 	}
 
-	if (!monitor_get_other_nodes(monitor, keeper->state.current_node_id,
-								 ANY_STATE, &(keeper->otherNodes)))
+	if (!keeper_refresh_other_nodes(keeper, forceCacheInvalidation))
```

Comment:
`keeper_refresh_other_nodes` does a bit more than `monitor_get_other_nodes`. It also updates pg_hba. Is this change in behavior on purpose?

Reply:
Yes I think it's a good idea to handle both concerns in one place. That's what we do with the list of other nodes after all. In most cases we don't expect changes in HBA to happen at this point, anyway.